yogstation/code/game/gamemodes/gangs/gangtool.dm

```diff
@@ -0,0 +1,260 @@
+//gangtool device
+/obj/item/device/gangtool
```

Comment:
Change all instances of `/obj/item/decive/gangtool` to `/obj/item/gangtool`. /tg/ got rid of /device a while back.